api/test/e2e/h3-data/h3-impact-map.spec.ts

```diff
@@ -267,6 +288,8 @@ describe('H3 Data Module (e2e) - Impact map', () => {
       expect(response.body.metadata).toEqual({
         quantiles: [617, 617, 617, 617, 617, 617, 617],
         unit: 'tonnes',
+        indicatorDataYear: 2020,
+        materialsH3DataYears: [],
```

Comment:
On top of: https://github.com/Vizzuality/landgriffon/pull/210/files#r784572397

I guess this is the use case for when we actually have matching years for materials. 
Should we add an empty array in this case?